pkg/kubelet/kubelet_test.go

```diff
@@ -1671,30 +1671,47 @@ func getNotReadyStatus(cName string) api.ContainerStatus {
 		Ready: false,
 	}
 }
+func getReadyCondition(status api.ConditionStatus, transitionTime util.Time, reason, message string) []api.PodCondition {
+	return []api.PodCondition{{
+		Type:               api.PodReady,
+		Status:             status,
+		LastTransitionTime: transitionTime,
```

Comment:
We are not comparing the ProbeTime in tests, since we cant inject or override time.Now().

One way to fix that would be to add a setNowForTest() method to util.Time and update Now() to return that time instead of the current time, if it is set. Then we can check that thats the time set in LastProbeTime.
But I wasnt sure if thats a good way. WDYT?